samba/CMakeLists.txt

```
@@ -450,5 +451,5 @@ add_custom_command(
 add_library(samba-static STATIC ${RPC_GEN_SRC} ${SAMBA_SRC} ${RPC_GEN_HEADERS})
 # Link against static heimdal-gssapi ldflags returned by pkg-config
 # Link against libhdb as well since it is missing from the list
-target_link_libraries (samba-static ${GSSAPI_STATIC_LDFLAGS} hdb)
+target_link_libraries (samba-static ${GSSAPI_STATIC_LDFLAGS} ${GNUTLS_STATIC_LDFLAGS} hdb )
```

```suggestion
target_link_libraries (samba-static ${GSSAPI_STATIC_LDFLAGS} ${GNUTLS_STATIC_LDFLAGS} hdb)
```

I guess the space is superfluous and should be dropped to keep the coding style consistent?